<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2021-06-23T22:59:19 CEST" name="Default suite" finished-at="2021-06-23T22:59:31 CEST" duration-ms="11974">
    <groups>
    </groups>
    <test started-at="2021-06-23T22:59:19 CEST" name="Default test" finished-at="2021-06-23T22:59:31 CEST" duration-ms="11974">
      <class name="tests.ProfileTest">
        <test-method is-config="true" signature="setup()[pri:0, instance:tests.ProfileTest@1b6e1eff]" started-at="2021-06-23T22:59:19 CEST" name="setup" finished-at="2021-06-23T22:59:22 CEST" duration-ms="2621" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="changeProfileImage()[pri:0, instance:tests.ProfileTest@1b6e1eff]" started-at="2021-06-23T22:59:22 CEST" name="changeProfileImage" finished-at="2021-06-23T22:59:28 CEST" duration-ms="6327" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Login Succesfull] but found [[[ChromeDriver: chrome on WINDOWS (b97347f6a890f87d28441cdd003ebda8)] -> xpath: //*[contains(@class, 'alert--success') or contains(@class, 'alert--danger')][contains(@style,'display: block')]]]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Login Succesfull] but found [[[ChromeDriver: chrome on WINDOWS (b97347f6a890f87d28441cdd003ebda8)] -> xpath: //*[contains(@class, 'alert--success') or contains(@class, 'alert--danger')][contains(@style,'display: block')]]]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:617)
at tests.ProfileTest.changeProfileImage(ProfileTest.java:65)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- changeProfileImage -->
        <test-method is-config="true" signature="takeScreenshotWhenFailure(org.testng.ITestResult)[pri:0, instance:tests.ProfileTest@1b6e1eff]" started-at="2021-06-23T22:59:28 CEST" name="takeScreenshotWhenFailure" finished-at="2021-06-23T22:59:31 CEST" duration-ms="2959" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=changeProfileImage status=FAILURE method=ProfileTest.changeProfileImage()[pri:0, instance:tests.ProfileTest@1b6e1eff] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- takeScreenshotWhenFailure -->
      </class> <!-- tests.ProfileTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
